features/guidance/turn-angles.feature

```diff
@@ -444,7 +444,7 @@ Feature: Simple Turns
 
        When I route I should get
             | waypoints | route          | turns                        |
-            | g,f       | turn,road,road | depart,turn left,arrive      |
+            | g,f       | turn,road      | depart,arrive                |
```

Comment:
actually closer to what we already see in osrm-backend for the respective intersection and, in my personal view, even the better result. It is a result of a small bugfix later on in `curve` handling